ethereum_history_api/circuits/lib/src/rlp/types_test.nr

```diff
@@ -49,40 +76,160 @@ mod rlp_fragment {
         fn invalid_value() {
             let invalid_value: [u8; 2] = [0x00, 0x00];
 
-            fragment.assert_eq_bytes(
+            fragment.assert_eq_bounded_vec(
                 "Field",
                 bytes,
                 BoundedVec { storage: invalid_value, len: invalid_value.len() }
             )
         }
     }
 
-    mod assert_eq_bytes32 {
+    mod assert_eq_null {
+        use crate::rlp::{decode::{STRING, LIST}, types::RlpFragment};
+        #[test]
+        fn success() {
+            let fragment = RlpFragment { offset: 0, length: 0, data_type: STRING };
+
+            fragment.assert_eq_null("Field")
+        }
+
+        #[test(should_fail_with = "Field: Invalid RLP type")]
+        fn invalid_type() {
+            let invalid_type_fragment = RlpFragment { offset: 0, length: 0, data_type: LIST };
```

Comment:
If you create a correct fragment outside once and just make copy&change here - it will be more readable